src/main/java/com/github/seratch/jslack/api/scim/ScimClientImpl.java

```diff
@@ -0,0 +1,36 @@
+package com.github.seratch.jslack.api.scim;
+
+import com.github.seratch.jslack.api.methods.SlackApiException;
+import com.github.seratch.jslack.api.scim.request.UsersDeleteRequest;
+import com.github.seratch.jslack.api.scim.response.UsersDeleteResponse;
+import com.github.seratch.jslack.common.http.SlackHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+import java.io.IOException;
+
+public class ScimClientImpl implements ScimClient {
+  private String endpointUrlPrefix = "https://api.slack.com/scim/v1/Users";
+
+  private final SlackHttpClient slackHttpClient;
+
+  public ScimClientImpl(SlackHttpClient slackHttpClient) {
+    this.slackHttpClient = slackHttpClient;
+  }
+
+  @Override
+  public void setEndpointUrlPrefix(String endpointUrlPrefix) {
+    this.endpointUrlPrefix = endpointUrlPrefix;
+  }
+
+  public UsersDeleteResponse delete(UsersDeleteRequest req) throws IOException, SlackApiException {
+    Request.Builder request = new Request.Builder().url(endpointUrlPrefix + "/" + req.getId()).addHeader("Authorization", "Bearer " + req.getToken());
+    return doPostForm(request, UsersDeleteResponse.class);
+  }
+
+  private <T> T doPostForm(Request.Builder requestBuilder, Class<T> clazz) throws IOException, SlackApiException {
```

Comment:
This method doesn't do a post method. How about `doDeleteRequest`?

Reply:
Also, you don't need `Request.Builder` here. Just giving a url should be enough.